microsoft/testsuites/core/timesync.py

```diff
@@ -167,41 +179,67 @@ def verify_timesync_unbind_clocksource(self, node: Node, log: Logger) -> None:
                     "lis_hyperv_clocksource_tsc_page",
                     "hyperv_clocksource",
                     "tsc",
+                    "Hyper-V-TSC",
                 ],
                 CpuArchitecture.ARM64: [
                     "arch_sys_counter",
+                    "ARM MPCore Timecounter",
                 ],
             }
+            cat = node.tools[Cat]
             lscpu = node.tools[Lscpu]
+            sysctl = node.tools[Sysctl]
```

Comment:
Declare tools in the corresponding if block. It avoids initializing unused tools.

Reply:
Moved to the corresponding blocks